fleetshard/pkg/central/reconciler/init_auth.go

```diff
@@ -163,9 +163,16 @@ func createAuthProviderRequest(central private.ManagedCentral) *storage.AuthProv
 		},
 		ClaimMappings: map[string]string{
 			"realm_access.roles": "groups",
+			"org_id":             "rhOrgId",
```

Comment:
We probably want to keep this in sync with what we have chosen for the migration script:
```suggestion
			"org_id":             "rh_org_id",
```